packages/components/note/src/Note.tsx

```diff
@@ -53,6 +53,10 @@ export type NoteInternalProps = CommonProps & {
    * @default false
    */
   withCloseButton?: boolean;
+  /**
+   * Expects any of the icon components to override the default Note variant icon
+   */
+  startIcon?: React.ReactElement;
```

Comment:
I was hesitant to use `icon` instead of `startIcon`, but to stay consistent with the API of other components I kept it.